templates/trustification/includes/_configure.tpl

```diff
@@ -0,0 +1,86 @@
+{{ define "rhtap.trustification.configure" }}
+- name: configure-trustification
+  image: quay.io/codeready-toolchain/oc-client-base:latest
+  command:
+    - /bin/bash
+    - -c
+    - |
+      set -o errexit
+      set -o nounset
+      set -o pipefail
+      set -x
```

Comment:
I find `set -x` to provide too much information to someone that is not very familiar with the content of the script.
I'd remove it and add a few nice `echo` statements to show where we are in the process.

Reply:
I think, for the time being, the `set-x` provides the right amount of information and as well doesn't hide errors. If I may ask, let's introduce this change on a future interaction, please.